.gitignore

```diff
@@ -1,6 +1,7 @@
 /ingress-operator
 
 tmp/
+/bin/*
```

Comment:
Curious, any particular reason to add `bin/` to `.gitignore`? Do any of the operator's make targets use a `bin/` directory?

Reply:
No, that's an oversight.  Thanks.